crates/topos-sequencer-subnet-runtime/src/proxy.rs

```diff
@@ -270,14 +270,16 @@ impl SubnetRuntimeProxy {
         subnet_client: &mut SubnetClient,
         cert: &Certificate,
         position: u64,
-    ) -> Result<String, Error> {
+    ) -> Result<Option<String>, Error> {
         debug!(
             "Pushing certificate with id {} to target subnet {}, tcc {}",
             cert.id, runtime_proxy_config.subnet_id, runtime_proxy_config.subnet_contract_address,
         );
         let receipt = subnet_client.push_certificate(cert, position).await?;
         debug!("Push certificate transaction receipt: {:?}", &receipt);
-        Ok("0x".to_string() + &hex::encode(receipt.transaction_hash))
+        let tx_hash =
+            receipt.map(|tx_receipt| "0x".to_string() + &hex::encode(tx_receipt.transaction_hash));
```

Comment:
Could we move this to a `Display` impl somewhere?

Reply:
Where do you suggest we should move it? `receipt` It is an `ethers` structure.